Homestead.yaml.example

```diff
@@ -0,0 +1,15 @@
+ip: 192.168.10.30
+memory: 2048
+cpus: 1
+provider: virtualbox
+authorize: ~/.ssh/id_rsa.pub
+keys:
+    - ~/.ssh/id_rsa
+folders:
+    - { map: ~/Sites/pastebin, to: /home/vagrant/pastebin }
```

Comment:
I agree 100% on the `Code` folder, which is why it's been removed on master, just haven't tagged a new version yet.